tests/test_meteodatalab/test_data_source.py

```diff
@@ -126,3 +126,31 @@ def test_retrieve_url(server):
     source = data_source.URLDataSource(urls=urls)
     for field in source.retrieve({"param": "T"}):
         assert field.metadata("shortName") == "T"
+
+
+# TODO: which one to use?
```

Comment:
I would say the one that is using real data because mocks can be annoying to maintain